electronics/ohms_law.py

```diff
@@ -0,0 +1,46 @@
+def ohms_law(voltage: float, current: float, resistance: float) -> float:
+    """
+    This function apply ohm's law, on any two given electrical values,
+    which can be voltage current resistance,
+    and then return name and value pair in the form of dictionary
+    >>> ohms_law(voltage=10, resistance=5, current=0)
+    {'current': 2.0}
+    >>> ohms_law(voltage=0, current=0, resistance=10)
+    Traceback (most recent call last):
+      File "<stdin>", line 14, in <module>
+    ValueError: Only one argument can be 0 at the time
+    >>> ohms_law(resistance=0, voltage=-10, current=1)
+    {'resistance': -10.0}
+    >>> ohms_law(voltage=0, current=-1.5, resistance=2)
+    {'voltage': -3.0}
+    """
+    if voltage == 0:
```

Comment:
Let's fast-fail.  This leaves the remaining logic simple and easy to reason about.
```suggestion
    if (voltage, current, resistance).count(0) != 1:
        raise ValueError("One and only one argument must be 0")
    if resistance < 0:
        raise ValueError("Resistance cannot be negative")
    if voltage == 0:
```